### PR TITLE
Impose maximum tag length

### LINKED ISSUE #164: [PE-D] Tag command bug (GUI)
When the tags are too long, it gets cut off at the end, making users unable to read the tag fully. You can consider allowing users to scroll horizontally to view the full tag (and expand the card horizontally when its necessary to fit in all information) 

Example: A SINGLE tag is too long

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: ruoyann/ped#10

### PR DESCRIPTION
**Pull request by kawaiigabrielneko**
Fixes #164

### CODE REVIEW

**Review by xianlinc — APPROVED** [review]
LGTM

**Comment by xianlinc** [comment]
You can increase the tag length, ill approve the PR first, so you can merge when you are done changing it.

File: src/main/java/seedu/insurancepal/model/tag/Tag.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
     public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
```
Comment (xianlinc) [inline src/main/java/seedu/insurancepal/model/tag/Tag.java:12]: Change message constraints to include character limit!

**Review by JoelChanZhiYang — APPROVED** [review]
LGTM

**Pull request merged**